src/widgets/buttons.ts

```diff
@@ -26,6 +27,16 @@ export type StatusAreaContext = {
   div?: HTMLElement
   dom?: HTMLDocument
 }
+enum ButtonType {
```

Comment:
I realized we already had two other cases in the examples both the cancel and continue buttons use the button widget so I think we should consider these styles as well.